lib/bloc/sentiment/sentiment_bloc.dart

```diff
@@ -22,11 +24,21 @@ class SentimentBloc extends Bloc<SentimentEvent, SentimentState> {
     try {
       emit(LoadingSentimentState());
       final positiveSentiment = await _sentimentRepo.getSentiment(event.query);
+      final negativeSentiment = positiveSentiment;
+      String rawCompletion = await openAIService.submitPrompt(event.query);
```

Comment:
Prefer using the keyword `final` instead of its type (here `String). Reason being its already implied that the method `openAIService.submitPrompt()` returns a string, plus `final` ensured that the variable is immutable and cannot be reassigned afterward.
`final rawCompletion = await openAIService.submitPrompt(event.query);`